stdlib/public/SDK/Foundation/NSCoder.swift

```diff
@@ -145,6 +145,13 @@ extension NSKeyedUnarchiver {
   }
 
   @nonobjc
+  @available(swift, obsoleted: 4)
+  @available(OSX 10.11, iOS 9.0, *)
+  public class func unarchiveTopLevelObjectWithData(_ data: Data) throws -> AnyObject? {
```

Comment:
I think at this point, we want this to be a non-source-breaking change, right? Here I'm just adding the versions of the methods which already exist; if we want to rename them, we'd have to do some API review